client/src/views/pages/Home.js

```diff
@@ -1,7 +1,35 @@
-const Home = () => (
-    <div>
-        <h1>Hello World!</h1>
-    </div>
-);
+import Navbar from "react-bootstrap/Navbar";
+import Container from "react-bootstrap/Container";
+import Button from "react-bootstrap/Button";
+import { handleLogout } from "@store/authentication";
+import { useDispatch } from "react-redux";
+import { useHistory } from "react-router-dom";
+
+function Home() {
+    const dispatch = useDispatch();
+    const history = useHistory();
+
+    const signOut = () => {
+        dispatch(handleLogout());
+        history.push("./login");
```

Comment:
I dont think you want a period before the route. Its a URL not a file. Same for below.

```suggestion
        history.push("/login");
```

Reply:
I see thanks!